app/core/config.py

```
@@ -16,52 +42,122 @@ class BaseConfig:
+    # CORS
+    BACKEND_CORS_ORIGINS: List[str] = ["*"]
 
     # database
-    DB_URL = "sqlite+aiosqlite:///:memory:"
-    SYNC_DB_URL = "sqlite:///:memory:"
+    DB: str = os.getenv("DB", "postgresql")
+    DB_USER: str = os.getenv("DB_USER", "postgres")
+    DB_PASSWORD: str = os.getenv("DB_PASSWORD", "password")
+    DB_HOST: str = os.getenv("DB_HOST", "localhost")
+    DB_PORT: str = os.getenv("DB_PORT", "5432")
+    DB_ENGINE: str = DB_ENGINE_MAPPER.get(DB, "postgresql")
```

Ruff (0.8.2)

Undefined name `DB_ENGINE_MAPPER`

(F821)